### PR TITLE
KK-715 | Fix User manager

### PR DESCRIPTION
**Pull request by tuomas777**
At least the `createsuperuser` management command was broken.

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -23,8 +23,13 @@ def possible_admins(self):
...
 
 
+# This is needed when using a custom User queryset
+class UserManager(OriginalUserManager.from_queryset(UserQuerySet)):
+    pass
+
+
 class User(AbstractUser):
-    objects = BaseUserManager.from_queryset(UserQuerySet)()
+    objects = UserManager()
```
Comment (charn) [inline users/models.py:32]: So I guess that separate class definition is required and `OriginalUserManager.from_queryset(UserQuerySet)` cannot be used here?

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit: I left a small comment, but as far as I know this looks awesome!

File: users/models.py (hunk 1 above)
Comment (tuomas777) [reply users/models.py:32]: Yeah it seems so, without the class at least `makemigrations` fails with

```
ValueError: Could not find manager UserManagerFromUserQuerySet in django.db.models.manager.
Please note that you need to inherit from managers you dynamically generated with 'from_queryset()'.
```

Comment (charn) [reply users/models.py:32]: Okay then, all good here! :+1:

**Pull request merged**